src/input_file.rst

```diff
@@ -767,6 +777,16 @@ Note that some tags, or entries for tags, are only used in certain ensembles (e.
 
   - Value 1: Double - Constant temperature of simulation in degrees Kelvin.
 
+  - Value 1: List of Doubles - A list of constant temperatures for simulations in degrees Kelvin.
+  .. code-block:: text
+
+        #################################
+        # SIMULATION CONDITION
+        #################################
+        Temperature   270.00    280.00    290.00    300.00 
```

Comment:
Provide another example for one temperature simulate to avoid user confusion.
You also need to add the same description for [Chempot](https://github.com/GOMC-WSU/Manual/pull/3/files#diff-e6ab4ebdafd50771470a27eb45f8e435R984), Fugacity, and Pressure. I assumed that Multisimulation feature is not limited to only multiple temperature simulation.

Reply:
![image](https://user-images.githubusercontent.com/39970712/72402467-623d8680-371d-11ea-88f9-cddfeab05f33.png)

![image](https://user-images.githubusercontent.com/39970712/72402514-7d0ffb00-371d-11ea-995f-190d1d7998f6.png)

![image](https://user-images.githubusercontent.com/39970712/72402604-a3ce3180-371d-11ea-8e12-f6aae382b78e.png)

![image](https://user-images.githubusercontent.com/39970712/72402617-b183b700-371d-11ea-9392-e9d03f08d90a.png)

Reply:
keep the fugacity fo `Si` and `O` 0.0.